### PR TITLE
Create mikasa.is-a.dev.json

### PR DESCRIPTION
**Pull request by 999shotoo**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/mikasa.is-a.dev.json
Comment (CuteDog5695) [inline domains/mikasa.is-a.dev.json:1]: remove the .is-a.dev part from the json name please

File: domains/mikasa.is-a.dev.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "",
+        "email": "mohitshedge5@gmail.com"
```
Comment (CuteDog5695) [inline domains/mikasa.is-a.dev.json:4]:
```suggestion
        "username": "shotoo999",
        "email": "mohitshedge5@gmail.com"
```

File: domains/mikasa.is-a.dev.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    },
+    "record": {
+        "CNAME": "superb-florentine-e0f2ef.netlify.app"
```
Comment (CuteDog5695) [inline domains/mikasa.is-a.dev.json:7]: we dont support Netlify at the moment (though we will soon)

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/mikasa.is-a.dev.json (hunk 2 above)
Comment (triach-rold) [reply domains/mikasa.is-a.dev.json:7]: Netlify links are unsupported. (Also, the website seems incomplete?)

**Comment by TobezDev** [comment]
@shotoo999
Your website is improperly setup. The Discord login redirects to `localhost:3000` rather than a static server. Would suggest looking into getting a small, cheapish VPS server to host your login requirement stuff.

**Comment by orxngc** [comment]
Closing as stale.

**Pull request closed without merging**